core/executor/src/wasm_executor.rs

```diff
@@ -367,12 +374,17 @@ struct SubstrateExternals;
 impl_wasm_host_interface! {
 	impl SubstrateExternals where context {
 		ext_malloc(size: WordSize) -> Pointer<u8> {
+			info!("DEBUG_CS_INSTRUMENTATION {}:{} TAG_Memory ext_malloc", file!(), line!());
```

Comment:
First, these functions are not used anymore.
Second, using `info!` to print debug information is not good.

Why not use grafana to collect this stuff?